### PR TITLE
Allow disabling Latin Keyboard input auto complete

### PR DESCRIPTION
**Pull request by HollowMan6**
As suggested in https://github.com/Igalia/wolvic/pull/1131#issuecomment-1895690337

### CODE REVIEW

File: app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java [hunk 1]
```diff
@@ -110,6 +110,7 @@ SettingsStore getInstance(final @NonNull Context aContext) {
     public final static String ENV_DEFAULT = "cyberpunk";
     public final static int MSAA_DEFAULT_LEVEL = 1;
     public final static boolean AUDIO_ENABLED = BuildConfig.FLAVOR_backend == "chromium";
+    public final static boolean LATIN_AUTO_COMPLETE_ENABLED = true;
```
Comment (svillar) [inline app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java:113]: I think we should have it disabled by default until we fix some of the issues that make it a bit cumbersome to use:
* speed: the longer the word the slower it becames, more than 5-6 characters and it starts to be sluggish
* space bar handling: blanks are not inserted correctly. The first click cancels de autocompletion but does not insert any character, you have to type twice to get the blank space
* insertions happening always at the end, start typing some word, then click in the middle of it to set the cursor in between some letters and then type another char, it's inserted at the end, not where the cursor is

I think I'm filling separate issues for these, in the meantime I'd suggest switching it off by default

**Review by svillar — CHANGES_REQUESTED** [review]

File: app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java (hunk 1 above)
Comment (svillar) [reply app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java:113]: Actually @felipeerias already filled #1183

Comment (svillar) [reply app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java:113]: Filed #1186 and #1187. Note that they're not blockers for this one, we should land this with autocompletion disabled by default and then enable it later once the behaviour is great.

Comment (HollowMan6) [reply app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java]: Sure, let's keep track of those issues.

**Review by svillar — APPROVED** [review]

**Pull request merged**